app/src/main/kotlin/com/mapbox/vision/teaser/MainActivity.kt

```diff
@@ -358,6 +376,18 @@ class MainActivity : AppCompatActivity(), ReplayModeFragment.OnSelectModeItemLis
         }
     }
 
+    private fun initRootTap() {
+        root.setOnClickListener {
+            if (visionManagerMode == Replay) {
+                if (playback_seek_bar_view.visibility == View.GONE) {
```

Comment:
toggleVisibleGone?

Reply:
yeah, will be changed with this issue https://github.com/mapbox/mapbox-vision-android-teaser/issues/191